code/game/objects/items/devices/scanners.dm

```diff
@@ -448,36 +448,41 @@ BREATH ANALYZER
+	to_chat(user, span("notice", "Health: [T.health]"))
+	to_chat(user, span("notice", "Nutrition: [T.nutrition]/[T.get_max_nutrition()]"))
+	if(T.nutrition < T.get_starve_nutrition())
+		to_chat(user, span("alert", "Warning: slime is starving!"))
 	else if (T.nutrition < T.get_hunger_nutrition())
-		user.show_message("<span class='warning'>Warning: slime is hungry</span>")
-	user.show_message("Electric change strength: [T.powerlevel]")
-	user.show_message("Health: [T.health]")
-	if (T.slime_mutation[4] == T.colour)
-		user.show_message("This slime does not evolve any further")
+		to_chat(user, span("warning", "Warning: slime is hungry!"))
+	to_chat(user, span("notice", "Electric charge strength: [T.powerlevel]"))
+	to_chat(user, span("notice", "Growth progress: [T.amount_grown]/10"))
+	if(T.cores > 1)
+		to_chat(user, span("warning", "Anomalous number of slime cores detected."))
+	else if(!T.cores)
+		to_chat(user, span("warning", "No slime cores detected."))
+	to_chat(user,span( "notice", "Genetic Information:"))
+	if(T.slime_mutation[4] == T.colour)
+		to_chat(user, span("warning", "This slime does not evolve any further."))
 	else
-		if (T.slime_mutation[3] == T.slime_mutation[4])
-			if (T.slime_mutation[2] == T.slime_mutation[1])
-				user.show_message(text("Possible mutation: []", T.slime_mutation[3]))
-				user.show_message("Genetic destability: [T.mutation_chance/2]% chance of mutation on splitting")
+		if(T.slime_mutation[3] == T.slime_mutation[4])
+			if(T.slime_mutation[2] == T.slime_mutation[1])
+				to_chat(user, span("notice", "Possible mutation: [T.slime_mutation[3]]"))
+				to_chat(user, span("notice", "Instability: [T.mutation_chance/2]% chance of mutation upon reproduction."))
 			else
-				user.show_message(text("Possible mutations: [], [], [] (x2)", T.slime_mutation[1], T.slime_mutation[2], T.slime_mutation[3]))
-				user.show_message("Genetic destability: [T.mutation_chance]% chance of mutation on splitting")
+				to_chat(user, span("notice", "Possible mutations: [T.slime_mutation[1]], [T.slime_mutation[2]], [T.slime_mutation[3]] (x2)"))
+				to_chat(user, span("notice", "Instability: [T.mutation_chance]% chance of mutation upon reproduction."))
 		else
-			user.show_message(text("Possible mutations: [], [], [], []", T.slime_mutation[1], T.slime_mutation[2], T.slime_mutation[3], T.slime_mutation[4]))
-			user.show_message("Genetic destability: [T.mutation_chance]% chance of mutation on splitting")
-	if (T.cores > 1)
-		user.show_message("Anomalious slime core amount detected")
-	user.show_message("Growth progress: [T.amount_grown]/10")
+			to_chat(user, span("notice", "Possible mutations: [T.slime_mutation[1]], [T.slime_mutation[2]], [T.slime_mutation[3]], [T.slime_mutation[4]]"))
+			to_chat(user, span("notice", "Instability: [T.mutation_chance]% chance of mutation upon reproduction."))
+	to_chat(user, span("notice", "**************************"))
```

Comment:
There has to be a way to do this with less copypaste.

Reply:
Dunno. Don't think there is.

Reply:
If ya know how, hit me up.

Reply:
```var/list/poss_mutations = uniquelist(T.slime_mutation)
var/mutation_message = ""

for(var/i in 1 to poss_mutations.len)
	mutation_message += "[poss_mutations[i]]"
	if(i != poss_mutations.len)
		mutation_message += ", "
	else
		mutation_message += "."

to_chat(user, span("notice"), "Possible mutations: [mutation_message]")
to_chat(user, span("notice"), "Instability: [T.mutation_chance][poss_mutations.len == 2 ? "/2" : ""]% chance of mutation upon reproduction.")```

Something like that? (Written assuming my earlier comment about number of mutations error is correct)